src/gui/clips/MidiClipView.cpp

```diff
@@ -523,7 +523,7 @@ void MidiClipView::paintEvent( QPaintEvent * )
 		p.scale(width(), height() - distanceToTop - 2 * notesBorder);
 
 		// set colour based on mute status
-		QColor noteFillColor = muted ? getMutedNoteFillColor().lighter(150)
+		QColor noteFillColor = muted ? getMutedNoteFillColor().lighter(200)
 									 : (c.lightness() > 127 ? getNoteFillColor().darker(400) : getNoteFillColor());
```

Comment:
If you change the c.lightness test to be a bit higher, even more than `(c.lightness() > 180` then the switch to a darker tone would come much later and the old themes will show like they did before unless you were going John Lennon post Yoko and all tracks were set to white.

Here's what it looks like with `c.lightness() > 200` and the colors are just around the switch.
![bright](https://github.com/LMMS/lmms/assets/6368949/912ba271-1e87-48b0-9eeb-caa1d7982966)
![bright2](https://github.com/LMMS/lmms/assets/6368949/3ce5f98e-bf8b-46f8-8628-c2a55c70a81d)

Reply:
PS. 200 is an example and maybe a bit much. Maybe ~170 is a good value?

Reply:
I've made a new commit addressing this issue. I felt that 175 was the value to settle at because it prioritizes the original look and feel with the lighter notes, but going beyond that results in readability issues with some clip color values (Example attached).
![Note-Readability](https://github.com/LMMS/lmms/assets/61636545/e3a83e7d-292a-4f31-9084-cee5a38c12a0)

Reply:
Oki doki!